pkg/utils/netlink_manager.go

```diff
@@ -0,0 +1,88 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright (C) 2023 Network Plumping Working Group
+// Copyright (C) 2023 Nordix Foundation.
+
+package utils
+
```

Comment:
duplication with https://github.com/opiproject/opi-evpn-bridge/blob/main/pkg/utils/netlink.go

Reply:
@glimchb in your netlink wrapper library that you want me to use each function there takes as parameter a context.

In this case here should I use an empty context right ? Like context.Background() ? Is this enough ?

Reply:
yes, `context.Background()` is perfectly valid

Reply:
@glimchb I think is not so trivial to just import that library and use it. The reason is that this library expects that there is some otel server that runs somewhere. If not then I get errors back related to the tracer.start(ctx) function
https://github.com/opiproject/opi-evpn-bridge/blob/main/pkg/utils/netlink.go#L50

We need to make the tracer.start code section optional in order to skip it if there is no otel service

Reply:
errors are not fatal, trying to send traces to otel collector should not fail... 

I also suggest to spin a `jaeger` like I do in *all* other OPI repos, example: https://github.com/opiproject/opi-evpn-bridge/blob/5440fb9f85ed80d74658412a0cf3ff352b8d29dd/docker-compose.yml#L221-L235

Reply:
@glimchb Even if that doesn't cause a fatal fail due to the fact that it waits until it timeouts (because there is no otel collector) make the calls extremely long and delayed. 

Also a CNI is just a binary that you throw in a K8s system and just works. Is highly unlikely to have CNI depend on an OTEL collector like jaeger. In most of the systems there is no Jaegger.

I think the implementation that we have today in the netlink wrapper is not good enough to just import it from a CNI. I believe we should make some changes to at least make the trace part optional.

WDYT ?

Reply:
Hello @glimchb can you please respond to my comment above ?

Reply:
done

https://github.com/opiproject/opi-evpn-bridge/pull/380